types/relay-compiler/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for relay-compiler 5.0
+// Project: https://relay.dev
+// Definitions by: n1ru4l <https://github.com/n1ru4l>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+import { GraphQLCompilerContext } from './lib/GraphQLCompilerContext';
+import { transformASTSchema } from './lib/ASTConvert';
+import * as Parser from './lib/RelayParser';
+import * as Printer from './lib/GraphQLIRPrinter';
+import ConsoleReporter from './lib/GraphQLConsoleReporter';
+import MultiReporter from './lib/GraphQLMultiReporter';
+
+export { GraphQLCompilerContext, transformASTSchema, Parser, Printer, ConsoleReporter, MultiReporter };
```

Comment:
It looks like ASTConvert is also exported directly (https://github.com/facebook/relay/blob/master/packages/relay-compiler/index.js#L118).

Reply:
done